### PR TITLE
CMR-6242 if Distribution format is just a comma, don't include empty …

### PR DESCRIPTION
**Pull request by eereiter**
…strings

### CODE REVIEW

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj [hunk 1]
```diff
@@ -4,6 +4,12 @@
   (:require
    [clojure.string :as string]))
 
+(defn remove-emtpy-strings
```
Comment (ygliuvt) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj]: use `defn-` to define private functions.

Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj]: fixed.

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj [hunk 2]
```diff
@@ -87,7 +93,7 @@
    string if ', ' exists or the input string if ', ' does not exist."
   [formats]
   (if (string/includes? formats ",")
-    (string/split formats #", *")
+    (remove-emtpy-strings (string/split formats #", *"))
```
Comment (mschmele) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj]: Nitpick: you could thread-first this to make it a little more readable
```suggestion
    (-> formats
        (string/split #", *") 
        remove-emtpy-strings)
```

Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj]: I have applied the changes.

**Pull request merged**